### DISCUSSION TITLE
Clean up CCU3 storage for update

### DISCUSSION THREAD

**Discussion by roel00** [OpenCCU/OpenCCU#2161] [🇺🇸 Q&A] [answered]
My CCU3 currently does not have enough storage to perform updates. How can I free up storage? I have no clue as to where to look for files that can be deleted.

**Comment by jens-maus** [comment, marked as answer]
When you are using RaspberryMatic on your CCU3 you probably missed to perform step 9 of the official documentation. See here:
https://github.com/jens-maus/RaspberryMatic/wiki/Installation-CCU3

In fact, in step 9 you were requested to perform a factory reset after you have installed RaspberryMatic so that it can utilize the full space of the 8GB sd card which unfortunately the official CCU3 firmware shipped by the vendor did not utilize.

So to solve your issue do the following:

1. Create a fresh backup via WebUI and download the *.sbk Backup file
2. Perform a factory reset ("System reset") via the WebUI and let the system startup freshly.
3. Check in the "Help" Page below the logout button if the localfs space is now listed with having enough disk space.
4. Restore the backup via the WebUI
5. After the final reboot everything should be back to normal but you should have significantly more disk space to be able to perform further updates.
